packages/terra-visually-hidden-text/docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Terra Visually Hidden Text
+
+Visually Hidden Text is text designed for screen readers that should not be visible to the UI. Some components such as badge will have a visual indication to note their hierarchy on the page, but that indication is lost when a screen reader is used on the web page. Visually Hidden text would allow your application to provide that additional context to a screen reader.
+
+Another reason for this component is when you want to produce more semantic markup for a screen reader only experience. On a screen reader, you may to provide a visually hidden header to allow for better readability. You also may need to provide different instructions for visual users vs non visual users. Finally, div's have a difficult understanding the aria-label attributes as well. In these instances, it's recommend to use visually hidden text.
```

Comment:
Need to add to word "want" in this phrase: On a screen reader, you may __want__ to provide a